Source/levels/drlg_l1.cpp

```diff
@@ -1003,44 +986,31 @@ void Substitution()
 
 void FillChambers()
 {
+	Point chamber1 { 14, 0 };
+	Point chamber3 { 14, 28 };
+	Point hall1 { 18, 12 };
+	Point hall2 { 18, 26 };
 	if (!VerticalLayout) {

```

Comment:
Pretty minor but you can structure these checks as positive by making the horizontal layout the default.

```suggestion
	Point chamber1 { 0, 14 };
	Point chamber3 { 28, 14 };
	Point hall1 { 12, 18 };
	Point hall2 { 26, 18 };
	if (VerticalLayout) {
```

Same could be done in FirstRoom with a bit more care.

Reply:
Sure why not